clar2wasm/src/standard/standard.wat

```diff
@@ -1447,21 +1499,36 @@
     )
 
     (func $stdlib.sha256-int (param $lo i64) (param $hi i64) (param $offset-result i32) (result i32 i32)
+        ;; This function calculates SHA-256 hash for 16-byte integer
+        ;; We can't store 16 byte integer in WASM, so we're using two variables, $lo and $hi
```

Comment:
Redundancy. This is what we do everywhere is this repo.

Reply:
Alright. I didn't know about other functions. Is this a prerequisite to go through other functions to understand this one? I'm not assuming the new person is a WASM ninja or something.

Reply:
Reading the ABI in the readme seems like a prerequisite.

Reply:
Oh. I missed that. Fixed